src/CacheInvalidator.php

```diff
@@ -20,6 +20,7 @@
 use FOS\HttpCache\ProxyClient\Invalidation\BanInterface;
 use FOS\HttpCache\ProxyClient\Invalidation\PurgeInterface;
 use FOS\HttpCache\ProxyClient\Invalidation\RefreshInterface;
+use FOS\HttpCache\Tag\TagHandler;
```

Comment:
i was pondering this. the reason to *not* use Handler was that i thought about the ObjectHandler we plan. that will likely use more than one class and even different strategies (tags vs banning subpaths). stuffing all of that into Handler feels weird. UserContext is also in its own namespace (though its not a "handler" so we can see that differently).

though i have to say, putting the ObjectHandler into Object sounds bizarre too. any good ideas for naming the two?